portafly/doc/ux_specs/product/create_product.md

```diff
@@ -0,0 +1,27 @@
+# Create Product
+
+[Back to index](../../index.md)
+
+* [Mockups](https://marvelapp.com/prototype/ee82j74/screen/70939425)
+* [Mockups JIRA ticket](https://issues.redhat.com/browse/THREESCALE-5548)
+* [Parent JIRA ticket](https://issues.redhat.com/browse/THREESCALE-5547)
+
+##### Form
+* Input fields
+  * Name > text input [required]
+    * Validation errors:
+      * Required field on form submit (if left blank)
+      * Name already exists on _onblur_
```

Comment:
Maybe is better to validate if name already exists also on submit.
That way we do the validation (for both cases) only once.
WDYT @josemigallas @didierofrivia ?

Reply:
@damianpm I was taking that for granted, but I will add it to the spec

Reply:
added in https://github.com/3scale/porta/pull/2030/commits/57c24917c89b46f6f05e65b634503c44d2421125